content/docs/use-cases/model-registry.md

```diff
@@ -53,7 +53,7 @@ list and transfer them within locations, wrap them as a local REST server, or
 even containerize and deploy them to cloud providers!
 
 This ecosystem of tools from [Iterative](https://iterative.ai/) brings your ML
-process into [GitOps]. This means you can manage and deliver ML models with
+process onto [GitOps]. This means you can manage and deliver ML models with
```

Comment:
alternative wording altogether:
- "... introduces GitOps into your ML process." 
- "... introduces GitOps framework into your ML process." 
- "... introduces GitOps methodology into your ML process."

Reply:
No strong opinion. Do you find the current wording suboptimal? If so we can go with your first suggestion.

Reply:
Same, no strong opinion, more of a personal preference.